toolkit/scripts/cherry_pick.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+function help {
+    echo "Cherry-pick commit to a specific branch and create Github PR"
+    echo "Usage:"
+    echo "-b TARGET_BRANCH -> target branch to cherry pick commit to"
+    echo "-c COMMIT_HASH"
+    echo "-l LOG_FILE -> log file to output conflicts in case cherry-pick fails, or to output URL to new PR"
+    echo "-o ORIGINAL_PR_URL -> original PR that triggers this script"
+    echo "-r REPOSITORY -> name of the repository"
+    echo "-t TITLE -> title of the original PR"
+}
+
+function cherry_pick {
+    commit_hash=$1
+    target_branch=$2
+    log_file=$3
+    pr_title=$4
+    original_pr_url=$5
+    repo=$6
+    tmp_branch="cherry-pick-$target_branch-$commit_hash"
+
+    echo "Commit hash = $commit_hash"
+    echo "Target branch = $target_branch"
+
+    git fetch --all
+    git checkout -b "$tmp_branch" origin/"$target_branch"
+
+    git cherry-pick -x "$commit_hash" || rc=$?
```

Comment:
Although I appreciate the `-x` here to add a ` (cherry picked from commit 86e3e69e2c10e8dd3c6a3beb6196824211f0ab97)`, I think it will only add value, if we are doing the `cherry-pick`s from a protected long-running branch, which is not transient; If we do a force-push to a branch from which we `cherry-pick`ed, the commit sha's will change and the information we captured in our commit will be invalid.

Are we planning to keep the `fasttrack` branches around without any force pushes ?